agenta-web/package.json

```diff
@@ -58,6 +58,6 @@
     "devDependencies": {
         "cypress": "^12.11.0",
         "node-mocks-http": "^1.12.2",
-        "prettier": "^3.0.0"
```

Comment:
my prettier version is different from to remote branch do i need to update this to 3.0.0